README.md

```diff
@@ -1,116 +1,59 @@
 # manta-picker
 
-This repo contains an under-development, early prototype of a standalone service that implements functionality similar to the mpicker tool in manta-muskie.  A draft RFD describing the motivation, requirements and approach for manta-picker is [here](https://github.com/joyent/rfd/tree/master/rfd/0170)
-
+The manta-picker is a Manta V2 service that provides an interface for retrieving a cached view  of the contents of the ```manta_storage``` bucket via the /poll REST endpoint.  The Rebalancer is the only current consumer of manta-picker.  Future versions of buckets-api will be modified to consume the manta-picker service.
```

Comment:
* Extra space between `view` and `of`
* Is the plan to still transition muskie (all picker consumers) to using this as well?  Perhaps we could just say that instead of specifying which repos are using it and which plan to use it.
* Is the picker Manta V2 specific?

Reply:
Removed the extra space.

Yes, the plan is to convert buckets-api (i.e. the mantav2 equivalent of muskie) to use the standalone picker.  That work is covered by MANTA-4821.  I don't think there are any other planned consumers of picker.

There's nothing V2-specific in the picker code.  But change to sdc-manta to make picker a first-class manta component was made to master after the mantav1 branch (see MANTA-4881).  If we want to use it on mantav1, we'll have to backport that change.

Reply:
I see.  That makes sense then.